src/murmur/Meta.cpp

```diff
@@ -440,15 +441,36 @@ void MetaParams::read(QString fname) {
 		qFatal("Qt without SSL Support");
 	}
 
-	QList<QSslCipher> pref;
-	foreach(QSslCipher c, QSslSocket::defaultCiphers()) {
-		if (c.usedBits() < 128)
-			continue;
-		pref << c;
+	qsCiphers = qsSettings->value("sslCiphers").toString().trimmed();
+
+	{
+		if (qsCiphers.isEmpty()) {
+			qsCiphers = MumbleSSL::defaultOpenSSLCipherString();
```

Comment:
Same as in client. Especially since you might want to query the default list it used from RPC.